### PR TITLE
Pull request: Fix signing on mac

### PR DESCRIPTION
**Pull request by ericpearson**
This fixes issue #177

I have tested this on Windows 7 and MacOS X 10.7.4 without any issues.

This should be merged to next as well.

### CODE REVIEW

**Comment by ericpearson** [comment]
@jkeshavarzi code review please

File: lib/signing-helper.js [hunk 1]
```diff
@@ -62,6 +62,8 @@ function execSigner(session, target, callback) {
 
     if (pkgrUtils.isWindows()) {
         script += ".bat";
+    } else {
+        script = cwd + script;
```
Comment (jamesjhedley) [inline lib/signing-helper.js]: I would add a quick comment as to why we are doing this.

**Comment by nukulb** [comment]
@ishneur - this needs to be tested on a MAC which does not have the NDK installed.

**Comment by jamesjhedley** [comment]
@ishneur We should also test on Windows and make sure we didn't break signing (should be good... but just to be safe). Also, when you are testing, make sure you have a space in the SDK folder name, so we can ensure we didn't reintroduce the previously fixed issue.

File: lib/signing-helper.js (hunk 1 above)
Comment (ericpearson) [inline lib/signing-helper.js]: done

**Comment by jamesjhedley** [comment]
r+

**Comment by ericpearson** [comment]
CI build is ready for testing

**Comment by ishneur** [comment]
Tested on a Mac after removing all NDKs from it. Signing works fine. Tried signing with and without a space in the packager folder name.

Also tested this on Windows 7 with/without space in the folder name and signing works fine.

**Comment by jamesjhedley** [comment]
Merged manually into rel6.

**Pull request closed without merging**